gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -887,6 +887,21 @@ public static boolean overlaps (Rectangle r1, Rectangle r2) {
 		return r1.overlaps(r2);
 	}
 
+	/**
+	 * @param r1x x of first {@link Rectangle}
+	 * @param r1y y of first {@link Rectangle}
+	 * @param r1w width of first {@link Rectangle}
+	 * @param r1h height of first {@link Rectangle}
+	 * @param r2x x of first {@link Rectangle}
```

Comment:
"second"

Nothing in the signature (except parameter names) suggest that this method deals with rectangles, I am in favor of renaming the method to something like `overlapRectangles`.